### PR TITLE
[JENKINS-62622] Do not throw exception with nonpublic plugins

### PR DESCRIPTION
**Pull request by daniel-beck**
See [JENKINS-62622](https://issues.jenkins-ci.org/browse/JENKINS-62622).

### Proposed changelog entries

* Do not log exceptions when browsing to the plugin manager while private source plugins are installed (regression in 2.240).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me, thanks for the patch! Taking the fact that the patch is trivial, I suggest to expedite it and to get it merged today so that it gets into the next weekly

I will merge it in 8 hours if there is no negative feedback.

**Review by amuniz — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Comment by daniel-beck** [comment]
FTR this appears to be a fairly minor issue per https://issues.jenkins-ci.org/browse/JENKINS-62622?focusedCommentId=392634&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-392634

**Comment by oleg-nenashev** [comment]
Anyway, I did not merge it in time :(

**Pull request merged**
